integration-tests/src/lib.rs

```diff
@@ -5,7 +5,9 @@ use crate::chains::{
 	paseo::{genesis::ED as PASEO_ED, PaseoRelayPallet},
 	pop_network::PopNetworkParaPallet,
 };
-use asset_hub_paseo_runtime::xcm_config::XcmConfig as AssetHubPaseoXcmConfig;
+// Note: using polkadot as stopgap until paseo updated to polkadot sdk v1.14.0
+use asset_hub_polkadot_runtime as asset_hub_runtime;
```

Comment:
Might be better to jsut use an export from asset_hub/src.rs if possible. That way the alias is defined in one place.